### DISCUSSION TITLE
node-red中如何获取小爱音箱语音内容

### DISCUSSION THREAD

**Discussion by 86727515** [XiaoMi/ha_xiaomi_home#283] [Q&A] [answered]
node-red中如何获取小爱音箱语音内容，如对着小爱说，小爱同学，打开空调。怎么node-red中获取“打开空调”

**Comment by milumet** [comment]
我也在研究这个。找不到

**Comment by JimmyKmi** [comment, marked as answer]
获取不到，参考设备spec小爱没有作文字上报

**Comment by Curisan** [comment]
同求

**Comment by qq173972819** [comment]
node-red执行完用小爱播报也没了

**Reply by yoder** [reply]
小爱播报有play_text，就是传入的格式要注意下["要播报的内容"]

**Reply by liangfyx** [reply]
怎么操作，可以截个图看看吗

**Reply by Teraflopst** [reply]
> 怎么操作，可以截个图看看吗

![Quicker_20241229_094934](https://github.com/user-attachments/assets/3569ea75-8a6c-44ba-b444-5005db2bda1d)

**Reply by 86727515** [reply]
这是让小爱播放文本，而不是获取小爱语音内容

**Reply by yoder** [reply]
他就是要播报啊

**Comment by milumet** [comment]
> node-red中如何获取小爱音箱语音内容，如对着小爱说，小爱同学，打开空调。怎么node-red中获取“打开空调”
很难受 现在没办法，直接用以前那些老方法好像

**Reply by hzcoolwind** [reply]
好消息是 停止小爱抢答可以用 暂停播放按钮来停止了

**Comment by weiyifeng73** [comment]
可以试试安装这个节点试试node-red-contrib-xiaoai-tts，看看能不能满足你的需要
